app/controller.py

```diff
@@ -130,12 +130,14 @@ def writeOrConfirmOverwrite(self):
       self.changeToConfirmOverwrite()
       return
     tb.fileWrite()
+    saveMessage = tb.message #Store the save message so it is not overwritten
```

Comment:
Please help me reproduce the situation that saveMessage fixing. I haven't hit it yet. I'm wondering if there's a deeper problem I should address, but I'd like to repro the issue to understand better.